src/components/Common/SavedAccountsView.tsx

```
@@ -36,21 +42,70 @@ export default function SavedAccountsView({
+                        <ListItem
+                            position="single"
+                            leading={
+                                <Image
+                                    src={MERCADO_PAGO}
+                                    alt="Mercado Pago"
+                                    width={32}
+                                    height={32}
+                                    className="size-8 min-w-8"
+                                />
+                            }
+                            title="Mercado Pago"
```

ESLint

[error] Copy passed as `title` must come from next-intl, not a literal — react/jsx-no-literals only inspects JSX children, so prop copy reaches every locale in English. Use t('…') from the right namespace. If this string is not copy (an id, a slug, a currency code), it does not need to be a full phrase.

(local/copy-props-from-catalog)